README.md

```diff
@@ -144,7 +144,7 @@ Ruby: SDK|Every Tuesday at 09:00 PT|[Google Doc](https://docs.google.com/documen
 Rust: SDK|N/A|[Google Doc](https://docs.google.com/document/d/1tGKuCsSnyT2McDncVJrMgg74_z8V06riWZa0Sr79I_4)|[Gitter](https://gitter.im/open-telemetry/opentelemetry-rust)|
 Swift: SDK|Every Thursday at 09:00 PT|[Google Doc](https://docs.google.com/document/d/1ShEFMywIV4LJcDYCNy41zkq8RR1sMq-tSvMBHngHcuk/edit?usp=sharing)|[Slack](https://cloud-native.slack.com/archives/C01NCHR19SB)||
 Communications (Website, etc.)|Every other week on Monday at 10:00 PT|[Google Doc](https://docs.google.com/document/d/1wW0jLldwXN8Nptq2xmgETGbGn9eWP8fitvD5njM-xZY)|[Slack](https://cloud-native.slack.com/archives/C02UN96HZH6)|
-End-User WG|Every other week on Thursday at 10:30 PT|[Google Doc](https://docs.google.com/document/d/1FqEo2PfHPP8iZX_W-C3YgXFUjhfwmAhoREUE8ZJPJfc/)||
+End-User WG|Every other week on Thursday at 10:30 PT|[Google Doc](https://docs.google.com/document/d/1e-UNZA3Tuno9b53RQbe--whUcO0VIXF3P81oXsrBK6g/edit#heading=h.ip90glf2wys2)|[Slack](https://cloud-native.slack.com/archives/C01RT3MSWGZ)|
```

Comment:
@melbgirl it looks like these are scheduled on the calendar at 10:00 PT instead of 10:30 PT, should I update the time in this doc before merging?

Reply:
@trask @melbgirl 10AM PT is the correct time.

Reply:
```suggestion
End-User WG|Every other week on Thursday at 10:00 PT|[Google Doc](https://docs.google.com/document/d/1e-UNZA3Tuno9b53RQbe--whUcO0VIXF3P81oXsrBK6g/edit#heading=h.ip90glf2wys2)|[Slack](https://cloud-native.slack.com/archives/C01RT3MSWGZ)|
```